src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php

```diff
@@ -84,8 +84,12 @@
             </div>
     	    </th>
           <td class="rightTD col-10">';
+                if (!empty($oField->oDefinition->sqlData['049_helptext'])) {
```

Comment:
Please do never use `empty` or the `!` operator (see https://docs.chameleon-system.de/contributing/best_practices.html#php-quirks) - can be replaced with empty string comparison here.

Reply:
oh, I saw this in many lines of the same file.
I changed it.